### PR TITLE
Update tests for max add docs count

### PR DESCRIPTION
**Pull request by vicilliar**
* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Feature

* **What is the current behavior?** (You can also link to an open issue here)
Tests fail due to max add docs limit added.

* **What is the new behavior (if this is a feature change)?**
- add docs batched test removed, as the server batch feature has been deprecated anyway.
- tests added to check different max doc counts and empty docs

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
no

* **Other information**:

### CODE REVIEW

File: tests/api_tests/cuda/test_cuda_add_documents.py [hunk 1]
```diff
@@ -88,24 +88,6 @@ def test_add_documents_with_ids_twice(self):
         self.client.index(self.index_name_1).add_documents([d2], device="cuda", non_tensor_fields=[])
         assert d2 == self.client.index(self.index_name_1).get_document("56")
 
-    def test_add_batched_documents(self):
```
Comment (pandu-k) [inline tests/api_tests/cuda/test_cuda_add_documents.py:91]: is this worth deleting? What about adding the `client_batch_size` arg to the add_docs call?

File: tests/api_tests/test_add_documents.py [hunk 2]
```diff
@@ -138,24 +138,6 @@ def test_add_documents_with_ids_twice(self):
         self.client.index(self.index_name_1).add_documents([d2], non_tensor_fields=[])
         assert d2 == self.client.index(self.index_name_1).get_document("56")
 
-    def test_add_batched_documents(self):
```
Comment (pandu-k) [inline tests/api_tests/test_add_documents.py:141]: ditto previous comment about using `client_batch_size`

**Review by pandu-k — APPROVED** [review]

**Review by pandu-k — CHANGES_REQUESTED** [review]
Sorry, accidentally clicked approve before. I've added some comments.

File: tests/api_tests/cuda/test_cuda_add_documents.py (hunk 1 above)
Comment (vicilliar) [reply tests/api_tests/cuda/test_cuda_add_documents.py:91]: I thought that adding `client_batch_size` would be better suited as a client test, but I can do it here as well.

**Review by pandu-k — APPROVED** [review]

**Pull request merged**